src/remixAppManager.js

```diff
@@ -262,6 +263,7 @@ export class RemixAppManager extends PluginEngine {
       icon: 'https://zokrates.blockchain-it.hr/zokrates.svg',
       location: 'sidePanel'
     }
+    const plugins = JSON.parse(await fetch(this.pluginsDirectory))
```

Comment:
`await fetch(path)` return a response object. You need to tranform it into a JSON object : 

```javascript
const res = await fetch(path);
const profiles = await res.json();
```